src/chipset/intel_i450kx.c

```diff
@@ -354,8 +346,8 @@ mc_write(int func, int addr, uint8_t val, void *priv)
     case 0x6d:
     case 0x6e:
     case 0x6f:
-        dev->mc_pci_conf[addr] = ((addr & 0x0f) % 2) ? val : (val & 7);
-        spd_write_drbs(dev->mc_pci_conf, 0x60, 0x6f, 1);
+        dev->mc_pci_conf[addr] = ((addr & 0x0f) % 2) ? 0 : (val & 0x7f);
```

Comment:
`((addr & 0x0f) % 2)` should probably be changed to `!(addr & 0x01)` for better clarity :P

Reply:
Sure (addr & 1) would be nice just modulo 2 came to my mind as first choice